prerequisites-check.sh

```diff
@@ -13,7 +16,22 @@ check_command_present git
 check_command_present docker
 
 YQ_VERSION="$(yq --version | ${SED} 's/^.* //g')"
-if [[ ${YQ_VERSION} != "3."* ]]; then
-  echo -e "${RED}yq vesion is ${YQ_VERSION}, version must be 3.*${NO_COLOUR}"
+# After version 3.3.1, yq --version sends the string to STDERR instead of STDOUT
+if [ -z "$YQ_VERSION" ]; then
+    YQ_VERSION="$(yq --version 2> >($GREP -Po '\d.\d.\d'))"
+fi
+
+yq_array=($(echo "$YQ_VERSION" | tr '.' '\n'))
```

Comment:
Do we still need to be able to handle both 3.3.1 as well as the previous versions on line 18? Or could we just remove it?

Reply:
The old version sends the version string to STDOUT so we need that line to capture that and tell the user to upgrade. The new version (3.3.1+) sends it to STDERR so we need to cover both to reliably check the version of `yq`.